modules/dm_Recycler/clients/base/views/record/record.js

```diff
@@ -1,9 +1,73 @@
 ({
     extendsFrom: 'RecordView',
 
+ //    events: {
+	// 	'click *' : 'preventClick'
+	// },
+
     initialize: function(options){
         console.log("dm_Recycler Record View Initialization.");
         this._super("initialize",[options]);
+    },
+
+    delegateButtonEvents: function() {
+        this._super("delegateButtonEvents");
+        this.context.on('button:restore_button:click', this.restoreRecord, this);
+    },
+
+    restoreRecord: function(){
+    	
+    	var self = this;
+
+    	//Disable buttons to avoide collision clicks
+    	this.toggleButtons(false);
+
+    	//Add loading message...
+    	app.alert.show('Recycler_restoreRecord', {
+            level: 'process'
+        });
+
+    	var url = app.api.buildURL('Recycler', this.model.id);
```

Comment:
Ok, so we weren't exactly on the same page. I see why... I call it Recycler, when in reality it is dm_Recycler, I just kind of leave the prefix off, because its easier to say that way. I'll run with this for now, but ill probably make an edit for clarification this weekend.